geoserverexplorer/qgis/sldadapter.py

```diff
@@ -46,11 +46,12 @@ def setScaleFactor():
     """Manage size scale factor basing if QGIS is able to manage or not SLD unit parameter (uom).
     """
     global SIZE_FACTOR
-    sldUomManaging = bool(QSettings().value("/GeoServer/Settings/QGIS/SldUomManaging", False, bool))
-    if sldUomManaging:
+    if pluginSetting("SldUomManaging"):
```

Comment:
@volaya I'm not sure if to leave the opportunity to the user to config this parameter of to check QGIS version to setup it automatically... e.g. version where UOM management is implementend (backported)
any opinion?
IMHO I would leave flexibility to the user

Reply:
yes, better to let the user configure it